data/mods/Magiclysm/Spells/classless.json

```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "id": "dark_sight",
+    "type": "SPELL",
+    "name": "Dark Sight",
+    "description": "Gives you the power to see in the dark",
+    "message": "Your eyes glow green for a moment. Now your sight can pierce the darkest shadows.",
```

Comment:
```suggestion
    "message": "Your eyes glow green for a moment.  Now your sight can pierce the darkest shadows.",
```

Reply:
Why two spaces?

Reply:
That's what we use for the in-game text: [Manual of style](/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md).

Reply:
hm. I guess there is no point in restating my question of why? :)

Reply:
@alkemann https://github.com/CleverRaven/Cataclysm-DDA/pull/5414#issuecomment-31611671